components/icons/DeleteIcon.tsx

```diff
@@ -0,0 +1,33 @@
+const DeleteIcon: React.FC = ({}) => (
+  <svg
+    width="20"
+    height="22"
+    viewBox="0 0 20 22"
+    stroke="#e4e4e4"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+  >
+    <path
+      fill-rule="evenodd"
```

Comment:
Haven't tested because not on computer, but assuming this will warn because it's not camel case. Same with others.
```suggestion
      fillRule="evenodd"
```

Reply:
yep they're totally doing that—fixing rn